### PR TITLE
APF-1 Data preparation and analysis

### PR DESCRIPTION
**Pull request by VladHolobyn**

### CODE REVIEW

File: notebooks/notebook.ipynb
Comment (VladHolobyn) [inline notebooks/notebook.ipynb:1]: image

Use missingo or simple `df.isna().sum()`

Comment (VladHolobyn) [inline notebooks/notebook.ipynb:1]: image

I'm not sure if there is a big difference, but I think `.mean()` would be better

**Review by 0DOlphin — APPROVED** [review]

File: notebooks/notebook.ipynb
Comment (VladHolobyn) [inline notebooks/notebook.ipynb:1]: For time series forecasting, we only use the date and the average price, so I don't think we need such a complex and detailed analysis:

image
image
image

image
image
image
image
image
image
image
image

Comment (VladHolobyn) [inline notebooks/notebook.ipynb:1]: image
image
image


Remove

Comment (VladHolobyn) [inline notebooks/notebook.ipynb:1]: image
image
image


The stationarity check will be done in the next tasks, so remove

Comment (VladHolobyn) [inline notebooks/notebook.ipynb:1]: Add train-test split

**Review by VladHolobyn — COMMENTED** [review]
Approved

**Pull request merged**